### ISSUE TITLE
some org markup ignore org headings height

### ISSUE THREAD

**Issue by nashamri** [bbatsov/solarized-emacs#201]
I'm not sure if this is an issue with org or the way emacs handle merging two faces.

![solarized](https://cloud.githubusercontent.com/assets/1207107/9287086/e0e1ef16-430d-11e5-96ab-50952a0c3f7b.png)

As shown in the pic, I have org headings vary in height and some of the org markup doesn't work well. the `<date>` and `+strikethrough+` just ignore the height property of the level 1 heading while `*bold*`, `/italic/` and `_underline_` work fine.

In short, all markup should inherit the height of the org headings.
